src/drive/web/modules/selection/SelectionBar.jsx

```diff
@@ -19,8 +19,22 @@ const mapDispatchToProps = dispatch => ({
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(({ selectionModeActive, ...rest }) => (
-  <div style={{ display: selectionModeActive ? 'inherit' : 'none' }}>
-    <SelectionBar {...rest} />
-  </div>
-))
+)(({ selectionModeActive, actions, ...rest }) => {
+  const convertedActions = DriveActionToSelectionBarAction(actions)
+  return (
+    <div style={{ display: selectionModeActive ? 'inherit' : 'none' }}>
```

Comment:
'inherit' could be undefined here I guess. I think the important bit is the 'display: none'.